packages/render-fidelity-tools/src/common.ts

```diff
@@ -170,11 +174,13 @@ export class ImageComparator {
           matched++;
         } else {
           mismatchingSum += delta;
+          mismatchingSquareSum += delta * delta;
         }
 
         const thresholdDelta = Math.max(0, delta - thresholdSquared);
 
         sum += thresholdDelta;
+        squareSum += thresholdDelta * thresholdDelta;
```

Comment:
Part of what I hate about the old metrics is this threshold, which is very arbitrary. Let's not follow their pattern here; instead let's only build a single metric that is independent of a threshold.